infra/lib/api-stack.ts

```diff
@@ -532,13 +533,14 @@ export class APIStack extends Stack {
   }
 
   private setupCdaToVisualization(ownProps: {
+    apiService: ecs_patterns.NetworkLoadBalancedFargateService;
```

Comment:
If could use `FargateService` it'd be nice, we can swap impl w/o code change

but NABD and do afterwards